node/src/Commands.ts

```diff
@@ -1984,3 +1984,34 @@ export function createZMPop(
 
     return createCommand(RequestType.ZMPop, args);
 }
+
+/**
+ * @internal
+ */
+export function createZRandMember(key: string): command_request.Command {
```

Comment:
why having different create functions? I think this could be a single function with optional parameters